src/renderer/component/common/tooltip.jsx

```diff
@@ -29,63 +29,6 @@ class ToolTip extends React.PureComponent<Props, State> {
     };
   }
 
-  componentDidMount() {
```

Comment:
this was slowing down the render. it wasn't really needed, more of a safe guard. It now requires manual testing to make sure a tooltip doesn't extend off the screen. I think this may be achievable with css, but haven't spent much time on it.